script/ot

```diff
@@ -40,7 +40,7 @@ if ($browse) {
 
 my @editor_args = editor_args_from_parsed_text($parsed);
 
-exec $ENV{EDITOR}, @editor_args;
+$ENV{EDITOR} ? exec $ENV{EDITOR}, @editor_args : die('nothing in EDITOR');
```

Comment:
Could we change this to something like `die('Please ensure your $EDITOR env var has been set')`?  I think that would be even clearer.